ASM/c/dungeon_info.c

```diff
@@ -293,12 +295,30 @@ void draw_dungeon_info(z64_disp_buf_t* db) {
 
         left += icon_size + padding;
 
-        // Draw dungeon names
-
+        // Draw the list of dungeons.
+        // Pre completed dungeons are grayed and crossed out.
         for (int i = 0; i < rows; i++) {
+            gDPPipeSync(db->p++);
             dungeon_entry_t* d = &(dungeons[i]);
+            int empty = CFG_DUNGEON_PRECOMPLETED[d->index];
             int top = start_top + ((icon_size + padding) * i) + 1;
-            text_print_size(db, d->name, left, top, font_width, font_height);
+            if (empty == 1) {
```

Comment:
```suggestion
            if (empty) {
```
Although `empty` should probably be boolean in the first place here, non-zero values are always "true" in C.